packages/yarnpkg-pnp/sources/types.ts

```diff
@@ -56,17 +54,13 @@ export type RuntimeState = {
   fallbackPool: Map<string, DependencyTarget>,
   ignorePattern: RegExp | null,
   packageLocationLengths: Array<number>,
-  packageLocatorsByLocations: Map<PortablePath, {locator: PhysicalPackageLocator, discardFromLookup: boolean} | null>;
+  packageLocatorsByLocations: Map<PortablePath, {locator: PhysicalPackageLocator, discardFromLookup: boolean}>;
   packageRegistry: PackageRegistry,
   dependencyTreeRoots: Array<PhysicalPackageLocator>,
 };
 
 // This is what the generation functions take as parameter
 export type PnpSettings = {
-  // Some locations that are not allowed to make a require call, period
-  // (usually the realpath of virtual packages)
-  blacklistedLocations?: Iterable<PortablePath>,
-
```

Comment:
I remember thinking that this setting was still useful for something, a few weeks ago, but I can't remember why... anyway, let's remove it for now and see if it comes up again later.